app/controllers/staff/transactions_controller.rb

```diff
@@ -108,7 +109,15 @@ def monetary_value
     end
   end
 
-  def fund_id
+  def hierarchy
```

Comment:
I think this needs renaming - when we're calling `@transaction.hierarchy` above, it looks like we're reaching into the *association* of it.  When I was prying in the console to figure out what was happening it took me a while to realise why calling `@transaction.hierachy` was returning `nil` rather than an `ActiveRecord::Relation`.

Reply:
Yeah, agree; I got myself in a big confused pickle while doing this bc of the naming. I think we might need another name for these.... concepts?